core/point/point.go

```diff
@@ -17,7 +43,38 @@ func New(x, y int64) *Point {
 }
 
 // X returns the x-value.
-func (p *Point) X() int64 { return p.x }
+func (pt *Point) X() int64 { return pt.x }
 
 // Y returns the y-value.
-func (p *Point) Y() int64 { return p.y }
+func (pt *Point) Y() int64 { return pt.y }
+
+// ToSGF converts a pointer-type (immutable) *Point
+// to an SGF Point (two letter string).
+// The returned value is 0-indexed.
+func (pt *Point) ToSGF() string {
+	sgfOut := ""
+	if (pt.X() <= 51) && (pt.Y() <= 51) {
+		sgfX := string(PointToSgfMap[pt.X()])
+		sgfY := string(PointToSgfMap[pt.Y()])
+		sgfOut = sgfX + sgfY
+	} else {
+		sgfOut = "--"
+	}
+	return sgfOut
+}
+
+// NewFromSGF converts an SGF point (two letter string)
+// to a pointer-type (immutable) *Point.
+func NewFromSGF(sgfPt string) *Point {
```

Comment:
Same comment here, I would return an error on bad data.

I can take this or you can.